src/App.php

```diff
@@ -157,6 +159,12 @@ class App
 
     public $templateClass = HtmlTemplate::class;
 
+    /** @var ResponseInterface */
+    private $response;
+
+    /** @var string Seed Response class  */
```

Comment:
seed !== class!
<==> array !== string-class

comment is thus even wrong, remove